src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt

```diff
@@ -121,11 +131,6 @@ def RunfilesEnvvar(module_space):
   if os.path.exists(runfiles):
     return ('RUNFILES_MANIFEST_FILE', runfiles)
 
-  # Look for the runfiles "input" manifest, argv[0] + ".runfiles/MANIFEST"
```

Comment:
What's the reason to remove this?

Reply:
I have restored the code I deleted.

I deleted it because it looked buggy to me.  It sets RUNFILES_DIR to
os.path.join(module_space, 'MANIFEST'), which is the name of a file,
not a directory.  All the other instances set RUNFILES_DIR to a directory
name.

Reply:
https://github.com/bazelbuild/bazel/blob/master/tools/python/runfiles/runfiles.py does not work when the environment variables are set improperly, so I imagine this branch is unreachable